### PR TITLE
Feature/JC19-255 Film page

### PR DESCRIPTION
**Pull request by artem-sotnichenko-saritasa**

### CODE REVIEW

File: vanilla/src/entities/specie/types.ts [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+
+  /** Primary key. */
+  readonly pk: number;
+}
+
+/** Specie DTO. */
+export interface SpecieDto extends SpecieDocument, FirebaseWrapper {}
+
+/** Specie model. */
+export interface Specie {
```
Comment (timofei-maximov-saritasa) [inline vanilla/src/entities/specie/types.ts:54]: `Specie` -> `Species` this word has only plural form

File: vanilla/src/entities/starship/types.ts [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+
+/** Starship model. */
+export interface Starship {
+
+  /**
+   * Speed or size in megalights.
+   *
+   * Megalight, standard unit of distance in space.
+   * */
+  readonly MGLT: string;
```
Comment (timofei-maximov-saritasa) [inline vanilla/src/entities/starship/types.ts:41]: Please, follow our naming convention [wiki](https://frontendwiki.saritasa.rocks/pages/development/principles/naming.html#avoid-contractions)

File: vanilla/src/base.css [hunk 3]
```diff
@@ -27,6 +27,7 @@
   --size-content-sm: var(--size-content-1);
```
Comment (golovachev-yaroslav) [inline vanilla/src/base.css:27]: `--size-content-1` is not obvious name, what about more informative name?

File: vanilla/src/film/display-film.ts [hunk 4]
```diff
@@ -0,0 +1,92 @@
...
+ * Create film element.
+ * @param film Film data.
+ */
+function createFilmEl(film: ConnectedFilm): HTMLDivElement {
+  const node = document.createElement('div');
+  node.classList.add('film');
+  node.innerHTML = `
+<h1 class="film__title">${film.title}</h1>
+<div class="film__producer film__meta-info meta-info">
+  <div class="meta-info__label">Opening crawl</div><div class="meta-info__text">${film.openingCrawl}</div>
```
Comment (golovachev-yaroslav) [inline vanilla/src/film/display-film.ts:14]: I think it would be better to use `h2` tag instead of `div`.

**Review by timofei-maximov-saritasa — APPROVED** [review]

**Review by golovachev-yaroslav — APPROVED** [review]

**Review by golovachev-yaroslav — APPROVED** [review]

**Review by golovachev-yaroslav — APPROVED** [review]

File: vanilla/src/base.css (hunk 3 above)
Comment (artem-sotnichenko-saritasa) [reply vanilla/src/base.css:27]: `--size-content-1` is from library, so to be informative, I mapped it into `--size-content-sm`.

**Review by dbitkovski — APPROVED** [review]

File: vanilla/src/entities/connected-film/index.ts [hunk 5]
```diff
@@ -0,0 +1,26 @@
...
+ */
+export async function getConnectedFilm(id: string): Promise<ConnectedFilm> {
+  const filmDto = await fetchFilmById(id);
+
+  const characterDtos = await fetchDocsByIds<CharacterDocument>('characters', filmDto.fields.characters);
+  const speciesDtos = await fetchDocsByIds<SpeciesDocument>('species', filmDto.fields.species);
+  const starshipDtos = await fetchDocsByIds<StarshipDocument>('starships', filmDto.fields.starships);
+  const vehicleDtos = await fetchDocsByIds<VehicleDocument>('vehicles', filmDto.fields.vehicles);
+  const planetDtos = await fetchDocsByIds<PlanetDocument>('planets', filmDto.fields.planets);
+
```
Comment (panov-aleksey-saritasa) [inline vanilla/src/entities/connected-film/index.ts:24]: I think you can use `Promise.all([...])` here in order to do all requests in parallel.

**Review by panov-aleksey-saritasa — APPROVED** [review]

**Pull request merged**